wagtail_streamfield_migration_toolkit/utils.py

```diff
@@ -1,6 +1,188 @@
+from collections import deque
```

Comment:
Are you using this any more? We should consider adding a linter like flake8 to the CI pipeline so things like this get caught